cmd/kubeadm/app/apis/kubeadm/v1alpha3/doc.go

```diff
@@ -16,12 +16,36 @@ limitations under the License.
 
 // Package v1alpha3 is the API (config file) for driving the kubeadm binary.
 // Some of these options are also available as command line flags, but
-// the preferred way to configure kubeadm is to pass a YAML file in with the
-// --config option.
+// the preferred way to configure kubeadm is to pass a single YAML file with
+// multiple configuration types in with the --config option.
 //
-// A fully populated example of the schema:
+// A fully populated example of a configuration file:
```

Comment:
Perhaps `s/a configuration/an init configuration` ?

Reply:
init configuration is a little misleading as we have an object called "InitConfiguration" and "ClusterConfiguration" when a cluster configuration is actually the useful thing to use during `kubeadm init`, but we could use both.

I think configuration file gets the meaning across while avoiding this confusion, but if you feel strongly I can change it.

Reply:
My idea here is not to confuse a user, that JoinConfiguration can be appended to this file. So if we manage to cover that case with some kind of word shuffling in either this or the previous paragraph, I am totally fine with it.